README.md

```diff
@@ -1,20 +1,78 @@
 # Create Custom SCW
 
-### Intall Dependency
 
+
+### How to set up the bundler?
+
+- First clone the repo `https://github.com/eth-infinitism/bundler.git`.
+
+- Now install all the modules in the root directory `budler/` by doing:
+```shell 
+yarn
+```
+
+- Then we need to build individual packages and resolve dependancy issues. For this first create a file `testnet-mnemonic.txt` and paste your private key inside it. Now in the terminal:
+```shell
+MNEMONIC_FILE=testnet-mnemonic.txt yarn preprocess
```

Comment:
We can use `MNEMONIC_FILE=<path-to-mnemonic>`

Reply:
fixed